### PR TITLE
add space for buggy kernel warning

### PR DESCRIPTION
**Pull request by lightmark**
add the missing space

### CODE REVIEW

File: env/io_posix.cc [hunk 1]
```diff
@@ -777,7 +777,7 @@ Status PosixWritableFile::Close() {
             file_stats.st_blksize !=
         file_stats.st_blocks / (file_stats.st_blksize / 512)) {
       fprintf(stderr,
-              "WARNING: Your kernel is buggy (<= 4.0.x) and does not free"
+              "WARNING: Your kernel is buggy (<= 4.0.x) and does not free "
               "preallocated blocks on truncate. Hacking around it, but you"
```
Comment (sagar0) [inline env/io_posix.cc]: A space is needed at the end of this line too.

File: env/io_posix.cc [hunk 2]
```diff
@@ -777,8 +777,8 @@ Status PosixWritableFile::Close() {
             file_stats.st_blksize !=
         file_stats.st_blocks / (file_stats.st_blksize / 512)) {
       fprintf(stderr,
-              "WARNING: Your kernel is buggy (<= 4.0.x) and does not free"
-              "preallocated blocks on truncate. Hacking around it, but you"
+              "WARNING: Your kernel is buggy (<= 4.0.x) and does not free "
+              "preallocated blocks on truncate. Hacking around it, but you "
```
Comment (siying) [inline env/io_posix.cc]: I thought you were adding a static so this is only printed out once.

Comment (siying) [inline env/io_posix.cc]: Ignore what I said. I didn't saw your buggy variable. Sorry.

File: env/io_posix.cc [hunk 3]
```diff
@@ -777,8 +777,8 @@ Status PosixWritableFile::Close() {
             file_stats.st_blksize !=
         file_stats.st_blocks / (file_stats.st_blksize / 512)) {
       fprintf(stderr,
-              "WARNING: Your kernel is buggy (<= 4.0.x) and does not free"
-              "preallocated blocks on truncate. Hacking around it, but you"
+              "WARNING: Your kernel is buggy (<= 4.0.x) and does not free "
```
Comment (siying) [inline env/io_posix.cc]: Can we say "may be" or something? Who knows whether there will be such a bug in the future kernel versions.

**Pull request closed without merging**